<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBoxFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>CSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XML</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonRead">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Считать данные</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>1001</width>
      <height>331</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabBus">
     <attribute name="title">
      <string>Автобус</string>
     </attribute>
     <widget class="QListView" name="listViewBus">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>201</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewBus">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>621</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTrain">
     <attribute name="title">
      <string>Поезд</string>
     </attribute>
     <widget class="QTableView" name="tableViewTrain">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>31</y>
        <width>681</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPlane">
     <attribute name="title">
      <string>Самолет</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QDateEdit" name="dateEditFrom">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditTo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEditFrom">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>HH:mm</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEditTo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>HH:mm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFrom">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFilter">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClosest">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти
ближайшие</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Откуда</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Куда</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправление</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Прибытие</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1122</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
